src/librustc_mir/borrow_check/type_check/mod.rs

```diff
@@ -1053,7 +1053,7 @@ impl<'a, 'tcx> TypeChecker<'a, 'tcx> {
     /// regions which are extracted and stored as having occurred at
     /// `locations`.
     ///
-    /// **Any `rustc::infer` operations that might generate region
+    /// **Any `rustc_middle::infer` operations that might generate region
```

Comment:
Again, I'm not sure `infer` is really there anymore.

Reply:
`rustc_middle::infer` exists, but it's only 3 small modules.